vidarrtest-regression.json.in

```diff
@@ -1,6 +1,7 @@
 [
     {
         "arguments": {
+            "hsMetrics.reference": "hg19",
```

Comment:
Since we are making a switch, there should be an update to CHANGELOG. I suggest we increment the second digit in the version as this is an addition of a new feature

Reply:
updated and tagged